Library/Homebrew/formula_installer.rb

```diff
@@ -156,7 +156,6 @@ def pour_bottle?(output_warning: false)
         opoo <<~EOS
           Building #{formula.full_name} from source as the bottle needs:
           - HOMEBREW_CELLAR: #{bottle.cellar} (yours is #{HOMEBREW_CELLAR})
-          - HOMEBREW_PREFIX: #{bottle.prefix} (yours is #{HOMEBREW_PREFIX})
```

Comment:
I think the prefix being correct is needed, no?

Reply:
`Bottle#prefix` is deprecated and always returns the default prefix.

Reply:
Gotcha (I did that, oops). I think I saw a user complaint about that somewhere but cannot find it. Probably need to just bake in the assumption that prefix is `Cellar/..`?

Reply:
Oh for the `HOMEBREW_REPOSITORY/Cellar` case? I'll try do something.

Reply:
Yeh, something like that.